modules/apps/frontend-taglib/frontend-taglib-form-navigator/src/main/java/com/liferay/frontend/taglib/form/navigator/internal/configuration/FormNavigatorEntryConfigurationRetriever.java

```diff
@@ -32,31 +32,29 @@
 @Component(service = FormNavigatorEntryConfigurationRetriever.class)
 public class FormNavigatorEntryConfigurationRetriever {
 
-	public Optional<List<String>> getFormNavigatorEntryKeys(
+	public List<String> getFormNavigatorEntryKeys(
 		String formNavigatorId, String categoryKey, String context) {
 
 		List<FormNavigatorEntryConfigurationParser>
 			formNavigatorEntryConfigurationParsers = ListUtil.fromCollection(
 				_serviceTrackerMap.getService(formNavigatorId));
 
-		Optional<List<String>> formNavigatorEntryKeysOptional =
-			Optional.empty();
+		List<String> formNavigatorEntryKeys = new ArrayList<>();
```

Comment:
@JanisZhang The original default was `Optional.empty()`, which should be mapped to `null`, not any real object.

Reply:
Hi @dantewang, I force-pushed and added null-check in its usages. Thanks!